logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/AvsClient.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.logic.data.call
+
+data class AvsClient(
+    val userId: String,
+    val clientId: String
+)
+
+data class AvsClientList(
+    val clients: List<AvsClient>
+) {
+    fun toJsonString(): String {
+        val start = "{"
+        var startClients = "clients:["
+
+        clients.forEachIndexed { index, avsClient ->
+            startClients += if (index > 0) "," else ""
+            startClients += "{userid: ${avsClient.userId},clientid: ${avsClient.clientId}}"
+        }
+
+        val endClients = "]"
+        val end = "}"
+
+        return "$start$startClients$endClients$end"
+    }
```

Comment:
```suggestion
    fun toJsonString(): String {
        val clientsListContent = clients.joinToString(separator = ",") {
            "{"userid": "${it.userId}","clientid": "${it.clientId}"}"
        }

        return """{"clients":[$clientsListContent]}"""
    }
```

Kotlin's stdlib to the rescue with `joinToString`.

Reply:
YES! Look a lot better and clean 👌🏻